examples/clearwater-blueprint.yaml

```diff
@@ -0,0 +1,193 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.5/types.yaml
+  - plugin.yaml
+
+inputs:
+
+  site_yaml_relative_path:
+    type: string
+    default: examples/clearwater-roles/site.yml
+
+  username:
+    type: string
+    default: vagrant
+
+  ellis_server_ip:
+    type: string
+    default: 11.0.0.10
+
+  ellis_private_key:
+    type: string
+
+  bono_server_ip:
+    type: string
+    default: 11.0.0.11
+
+  bono_private_key:
+    type: string
+
+  sprout_server_ip:
+    type: string
+    default: 11.0.0.12
+
+  sprout_private_key:
+    type: string
+
+  homer_server_ip:
+    type: string
+    default: 11.0.0.13
+
+  homer_private_key:
+    type: string
+
+  homestead_server_ip:
+    type: string
+    default: 11.0.0.14
+
+  homestead_private_key:
+    type: string
+
+  ralf_server_ip:
+    type: string
+    default: 11.0.0.15
+
+  ralf_private_key:
+    type: string
+
+  bind_server_ip:
+    type: string
+    default: 11.0.0.16
+
+  bind_private_key:
+    type: string
+
+node_types:
+
+  testbed:
+    derived_from: cloudify.nodes.Compute
+
+  ellis:
+    derived_from: testbed
+    properties:
+      agent_config:
+        default:
+          install_method: none
+          key: { get_input: ellis_private_key }
+          user: { get_input: username }
+
+  bono:
+    derived_from: testbed
+    properties:
+      agent_config:
+        default:
+          install_method: none
+          key: { get_input: ellis_private_key }
```

Comment:
Are you really need to use `ellis_private_key` or it should be `bono_private_key`